pages/blog/getting-the-best-out-of-your-graphql-client-by-using-relay-patterns-through-graphql-code-generator.mdx

```diff
@@ -0,0 +1,212 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title:
+    'Getting the best out of your GraphQL client by using Relay patterns through GraphQL Code Generator',
+  author: 'arda',
+  tags: ['graphql'],
+  date: '2022-08-01',
+  description: 'TBA',
+  image:
+    '/blog-assets/getting-the-best-out-of-your-graphql-client-by-using-relay-patterns-through-graphql-code-generator/cover.jpg'
+}
+
+export default withArticle(meta)
+
+# Getting the best out of your GraphQL client by using Relay patterns through GraphQL Code Generator
+
+There is no doubt that Relay is the most advanced JavaScript GraphQL Client that is out there. However, at the same time, the learning curve and adoption is yet very sparse compared to other popular alternatives such as Apollo Client, Urql, or GraphQL Request.
+
+As a consequence, many people don’t even know about all the benefits and patterns used within (and so far being exclusive to ) the relay-compiler, the GraphQL Code Generator counter-part tailored for Relay applications.
```

Comment:
@n1ru4l, regarding the discussion on the confusing use of "GraphQL Code Generator" here:

> I’m not sure we need to mention the relay compiler here, we can stick on Relay right? (since we mainly talk about Fragments)

Reply:
I think I would skip that whole part around counter-part and write it as separate sentence later